.github/workflows/testing.yml

```
@@ -27,18 +37,42 @@ jobs:
       - name: Clean up clusters
         run: make clean-clusters
 
-  compile:
-    name: Compile
+  make-clean-clusters:
+    name: Make clean-clusters
```

I prefer the names to reflect the purpose as phrases rather than the internal make target. 

```suggestion
    name: Clean up clusters
```